starknet_devnet/blueprints/rpc/structures/payloads.py

```diff
@@ -1006,14 +1006,48 @@ def rpc_map_traces(
     return result
 
 
+def gateway_to_rpc_invocation(invocation_dict: dict) -> Dict[str, Dict]:
+    """
+    Convert function invocation result from gateway to RPC.
+    """
+    if invocation_dict is None:
+        return invocation_dict
+
+    # All unnecessary properties should be removed except this mentioned here.
+    accepted_properties = {
```

Comment:
According to the specification, shouldn't these properties be split into two objects: an object of type FUNCTION_CALL (contract_address, entry_point_selector, calldata) and another one with properties caller_address, class_hash, entry_point_type, result, calls, events, messages)

Reply:
My understanding of `allOf` was wrong. I means that there should be one object conforming to both subschemas.
https://json-schema.org/understanding-json-schema/reference/combining.html